lib/client.js

```diff
@@ -194,6 +206,27 @@ Client.prototype.getWalletDetails = function (ip, _opts) {
   return this._request('GetWalletDetails', ip, opts).get('WALLET');
 };
 
+Client.prototype.getWalletDetailsBatch = function (ip, _opts) {
+  var opts = _.merge({
+    version: '1.1',
```

Comment:
Shouldn't we use the [latest version `2.0`](https://documentation.lemonway.com/reference/getwalletdetailsbatch#response-output-description) to get more details?